### ISSUE TITLE
A11y_Visual Studio Code Web Extensions_Github Pull request_Create Pull request_TextSpacing: Text spacing does not adapt for 'Create Pull request' section.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/vscode-pull-request-github#4238]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#GH_Visual Studio Code Web Extensions_Nov2022;#Visual Studio Code Web Extensions ;#ChromiumEdge;#DesktopWeb;#A11ySev2;#Win11;#WCAG1.4.12;#TextSpacing;#A11yWCAG2.1;#Closed;

## Environment Details:
Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Windows Version: Win11
Edge version:107.0.1418.56

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Open https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
3. tab to 'Github Pull request' and press enter.
4. tab to 'Create Pull request' and press enter.
5. Apply text spacing and check issue.

## Actual:
Text spacing does not adapt for 'Create Pull request' section

**Similar issue observed in 'Github_Description' scenario.**

## Expected:
Text spacing should be applied for 'Create Pull request' section.

## User Impact:
Users with Visual problem may find it difficult to read the content.

## Attachment:

![TextSpacing](https://user-images.githubusercontent.com/85863999/203701350-3d61ee2a-c835-48c2-929f-298bdecf9b85.gif)

**Comment by sspatil123** [comment]
After applying stylus page is adopting text spacing. Hence closing bug. 
![4238_NotRepro](https://user-images.githubusercontent.com/85863999/206613611-ba44810d-7386-4484-8300-f2022de702ab.png)
![NotRepro_4238](https://user-images.githubusercontent.com/85863999/206613622-06241aa6-e123-4a7a-9210-3f248d05f149.png)
